package.json

```
@@ -25,7 +25,7 @@
     },
     "devDependencies": {
         "@definitelytyped/definitions-parser": "latest",
-        "@definitelytyped/dtslint": "^0.0.159",
+        "@definitelytyped/dtslint": "^0.0.162",
```

```suggestion
        "@definitelytyped/dtslint": "latest",
```

This shouldn't actually have a version at all, it should be `latest` like the other. It looks like it mistakenly changed (again) in #65130.